### PR TITLE
Add the image-template on homepage

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added the image-template to the project and used it for all the images on the homepage.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Open your network tab in the inspector
- See that images file size has dropped from **370KB** to **60KB**

### CODE REVIEW

**Comment by barrymcgee** [comment]
@anthonydillon You'll need to tell linkchecker to ignore Cloudindary tags etc - see jaas.ai example https://github.com/canonical-web-and-design/jaas.ai/blob/master/.circleci/config.yml#L35

**Comment by anthonydillon** [comment]
@barrymcgee morning, this one should be ready again.

File: templates/base_index.html [hunk 1]
```diff
@@ -101,7 +101,15 @@ <h2 class="p-heading--three u-no-max-width">
     <section class="p-strip is-deep u-image-position is-bordered">
       <div class="row">
         <div class="col-7">
-          <img src="https://assets.ubuntu.com/v1/0bfbd13b-cloud_section_bundle.svg?h=460" class="u-image-position--top u-hide--small" alt="" height="460" style="height: 100%; right: 47%;" />
+          {{
```
Comment (barrymcgee) [inline templates/base_index.html]: This is a different image - expected?

File: templates/base_index.html [hunk 2]
```diff
@@ -101,7 +101,15 @@ <h2 class="p-heading--three u-no-max-width">
...
+            image(
+              url="https://assets.ubuntu.com/v1/450d7c2f-openstack-hero.svg",
+              alt="",
```
Comment (barrymcgee) [inline templates/base_index.html]: Why blank alt?

File: templates/base_index.html [hunk 3]
```diff
@@ -118,7 +126,16 @@ <h3><a href="/internet-of-things">The fastest way to go from development to prod
           <p>Learn about how Ubuntu Core and snaps can help you build your connected devices.</p>
         </div>
         <div class="col-7 col-start-large-6 u-align--center">
-          <img class="u-hide--small" src="https://assets.ubuntu.com/v1/808a4e5b-iot.png?h=300" height="300" alt="" />
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/808a4e5b-iot.png",
+              alt="",
```
Comment (barrymcgee) [inline templates/base_index.html]: Blanka alt?

File: templates/engage/openstack-telco-clouds.html [hunk 4]
```diff
@@ -29,7 +29,7 @@
     <p>Discover why Ubuntu OpenStack enables Telcos to maximise margins, reduce the costs and risks of introducing new value-added services and speed up time-to-revenue.</p>
 
             <p>Deutsche Telekom is developing and deploying many new cloud applications on Ubuntu OpenStack as part of its secure and highly scaleable public cloud Business Marketplace ecosystem for SMB customers.</p>
-            <img src="http://assets.ubuntu.com/sites/ubuntu/1184/u/img/logos/logo-deutsche-telekom.png" 
+            <img src="http://assets.ubuntu.com/sites/ubuntu/1184/u/img/logos/logo-deutsche-telekom.png"
```
Comment (barrymcgee) [inline templates/engage/openstack-telco-clouds.html]: Vector logo available here: https://worldvectorlogo.com/logo/deutsche-telekom-2

File: templates/engage/shared/_header.html [hunk 5]
```diff
@@ -20,7 +20,7 @@ <h1 {% if subtitle is none and date is none %}class="u-no-margin--bottom"{% endi
       </div>
     </div>
     {% if image %}<div class="col-5 u-hide--small u-align--center u-vertically-center">
-      <img src="{% if 'http' not in image %}https://assets.ubuntu.com/v1/{% endif %}{{ image }}" width="413" style="max-height: 410px; max-width: 250px;" alt="image for {{ title }}" >
+      <img src="{% if 'http' not in header_image %}https://assets.ubuntu.com/v1/{% endif %}{{ header_image }}" width="413" style="max-height: 410px; max-width: 250px;" alt="image for {{ title }}" >
```
Comment (barrymcgee) [inline templates/engage/shared/_header.html]: The width here is much greater than the inline style restricting the width.

File: templates/base_index.html [hunk 6]
```diff
@@ -127,7 +144,16 @@ <h3><a href="/internet-of-things">The fastest way to go from development to prod
     <section class="p-strip is-deep is-bordered">
       <div class="row u-equal-height u-vertically-center">
         <div class="col-7">
-          <img src="https://assets.ubuntu.com/v1/668136e7-containers.svg" class="u-hide--small" style="height: 85%;" height="374" alt="" />
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/668136e7-containers.svg",
+              alt="",
```
Comment (barrymcgee) [inline templates/base_index.html]: Blank alt?

File: templates/base_index.html [hunk 7]
```diff
@@ -144,7 +170,16 @@ <h3><a href="/desktop">Fast, secure and simple, Ubuntu powers millions of PCs wo
           <p>Download the latest version of Ubuntu, for desktop PCs and laptops.</p>
         </div>
         <div class="col-7 col-start-large-6 u-align--center">
-          <img class="u-hide--small" src="https://assets.ubuntu.com/v1/c6504e94-Dell_XPS_Laptop_Front-Desktop.png?h=350" height="350" alt="" />
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/c6504e94-Dell_XPS_Laptop_Front-Desktop.png",
+              alt="",
```
Comment (barrymcgee) [inline templates/base_index.html]: Blank alt?

File: templates/base_index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/base_index.html]: Yeah, the Juju one was complicated with image position but I asked Spencer for a more simpler image.

File: templates/base_index.html (hunk 2 above)
Comment (anthonydillon) [reply templates/base_index.html]: Just matched the current alt. Also its not that useful for accessibility. Might be worth removing altogether in the future.

File: templates/engage/shared/_header.html (hunk 5 above)
Comment (anthonydillon) [reply templates/engage/shared/_header.html]: I just changed this to work with the new module. There are likely issues.

File: templates/base_index.html (hunk 2 above)
Comment (barrymcgee) [reply templates/base_index.html]: If you think they're all decorative, fair enough but don't remove the alt as that's against WebAIM recommendations. https://webaim.org/techniques/alttext/

**Pull request merged**